src/traits/PrivacyTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace luya\privacy\traits;
+
+use Yii;
+use yii\web\Cookie;
+
+/**
+ * Trait PrivacyTrait
+ * 
+ * The privacy trait helps getting and setting the privacy policy state.
+ * 
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.0
+ */
+trait PrivacyTrait
```

Comment:
use a CONST for the cookie name defintion like

const PRIVACY_COOKIE_NAME = '_privacyPolicy'

Reply:
Not possible in trait.. So where do you want that?
Changed it to static for the moment..

Reply:
ok